### ISSUE TITLE
Need more text about RPubs, GitHub, and Shiny

### ISSUE THREAD

**Issue by ToonTalk** [ox-it/LiveData_htmlwidgets#33]
In https://github.com/ox-it/OxfordIDN_htmlwidgets/blob/gh-pages/index.Rmd

> RPubs, GitHub Pages or to include such interactive data visualisations within a Shiny app

I worry that some readers won't know what RPubs, GitHub, and Shiny are. A brief explanation with links to more info might be best.

**Comment by charliejhadley** [comment]
Good point - do need some more high level "this is what the RStudio ecosystem provides"
